src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -225,7 +225,7 @@ class TezosDatabaseOperationsTest
             val generatedUpdateRows =
             operation
               .convertToA[List, Tables.BalanceUpdatesRow]
-              .map(_.copy(sourceId = Some(opRow.operationId)))
+              .map(_.copy(sourceId = Some(opRow.operationId), operationGroupHash = Some(opRow.operationGroupHash)))
```

Comment:
Needed?

Reply:
See above. This is the quick fix that works, in running Lorre and in tests